lib/components/InputBase/NotchedBase.tsx

```diff
@@ -65,3 +67,11 @@ export const NotchedBase: FunctionComponent<IProps> = ({
 		</div>
 	);
 };
+
+function getNotchedComputedWidthFotWidth(width: number): number {
```

Comment:
Typo in `getNotchedComputedWidthFotWidth `

Reply:
:100: good spot!